components/script/dom/htmlformelement.rs

```diff
@@ -308,12 +308,17 @@ impl HTMLFormElement {
                         DispositionParam::Filename(Charset::Ext(String::from(charset.clone())),
                                                    None,
                                                    f.name().clone().into()));
+                    /// XXX: unwrap
```

Comment:
replace it with an expect

though in this case `unwrap_or(text/plain)` might be better?